ml-agents/mlagents/trainers/tests/test_learn.py

```diff
@@ -75,17 +76,24 @@ def test_run_training(
                 learn.run_training(0, options)
                 mock_init.assert_called_once_with(
                     trainer_factory_mock.return_value,
-                    "results/ppo",
+                    os.path.join("results", "ppo"),
```

Comment:
I don't think this is a bad change, but does it need to happen in this PR? Does this test fail on a certain platform (Windows?) without it?

Reply:
Yes, it fails in Windows (I am using Windows)

Reply:
OK. I'll make a note for us to run pytest on Windows too.